core/storage/user/gae_models.py

```diff
@@ -304,6 +316,7 @@ def get_user_id_migration_policy():
         """
         return base_models.USER_ID_MIGRATION_POLICY.COPY
 
+
```

Comment:
Drop newline here. Only one newline between adjacent methods in a class.

Reply:
Done